samcli/commands/local/lib/sam_function_provider.py

```diff
@@ -258,12 +217,17 @@ def _parse_layer_info(list_of_layers, resources):
                 codeuri = None
 
                 if resource_type == SamFunctionProvider._LAMBDA_LAYER:
-                    codeuri = SamFunctionProvider._extract_lambda_function_code(layer_properties, "Content")
+                    codeuri = SamFunctionCodeProvider.extract_lambda_function_code(
```

Comment:
We should have more generic names. This function is centered around Layers but we are extracting function code.